core/tests/performance_framework/process_data.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script to process page loading stats or HTTP Archive(also referred to as HAR)
+data and/or timing stats to obtain stats/metrics for the corresponding page.
+
+We need selenium + browsermob-proxy to capture page loading session
+(or HTTP Archive), to obtain page size stats. However, owing to the sluggish
```

Comment:
Please don't interchangeable use 'session data' and HTTP archive since they aren't the same thing. As I understand, an HTTP archive is simple a file format which contains session data. We actually don't care at all about the HAR file itself in this code, so it's not particularly noteworthy to refer to it. Explaining where this data comes from once (at the top docstring) seems reasonable, but knowing the data originally came from a HAR file doesn't provide necessary context anywhere else for understanding the code. I suggest not reinforcing this association since it might spread misinformation.